deps/rabbitmq_stream/docs/PROTOCOL.adoc

```diff
@@ -394,7 +397,9 @@ Deliver => Key Version SubscriptionId OsirisChunk
   ChunkCrc => int32
   DataLength => uint32
   TrailerLength => uint32
-  Reserved => unit32 // unused 4 bytes
+  BloomSize => uint8 // size of bloom filter data
+  Reserved => unit24 // unused 3 bytes
+  BloomFilterData => [uint8] // bloom filter data (no int32 for the size of this array, the size is defined by BloomSize
```

Comment:
@michaelklishin Unfortunately, this seems to be invalid.

I receive `BloomSize` default value (16), but `BloomFilterData` is missing.

Looking at Java client (24cc76a51), class `ServerFrameHandler` ignores `BloomSize`, and does not read `BloomFilterData` indeed.

Reply:
https://github.com/rabbitmq/rabbitmq-server/pull/11533/

Reply:
That is right: Osiris does not send the bloom filter data to client applications. Sorry for the confusion.